recipe/meta.yaml

```diff
@@ -43,9 +50,7 @@ test:
     - pip check
     - tqdm --help
     - tqdm -v
```

Comment:
Might be good to compare to the {{ version }}:
tqdm -v | grep -q {{ version }}

as I noticed in one of the packages it was giving incorrect version of 0.0.0, without raising an error.